packages/jest-environment-node/src/index.ts

```diff
@@ -82,6 +82,13 @@ class NodeEnvironment implements JestEnvironment<Timer> {
     if (typeof performance !== 'undefined') {
       global.performance = performance;
     }
+    if (
+      typeof atob !== 'undefined' &&
```

Comment:
Should we add a comment here that it is available in node 16 like it was done above?